docs/web/tutorial-common-lifecycle-hooks.mdx

```diff
@@ -0,0 +1,156 @@
+
+## Application Initialized
+
+This event gets executed when Web has initialized. Commands or Workflows used here must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading in Designer.
+
+# Components > Map > Events >
+
+## Map Initialized
+
+This is an event that gets triggered when the map is initialized. The map is initialized when it is loaded for the first time, a user signs in or out, or in Mobile when a map area is activated or deactivated. This can be useful for displaying a Workflow that presents UI such as a form or for a Workflow that depends upon map data.
+
+# Services > Launch Link Workflows
+
+Using the Launch Link service you can add and remove Workflows to the app that can be optionally specified and run by a URL parameter. The launch link Workflow is run at the same time as the Map Initialized Event. This can be useful for passing in information such as an asset id and then using a Workflow to have Web start with that asset already centered and selected.
+
+<img
+    src={useBaseUrl("img/web-workflow-lifecycle-launch-link-3.png")}
+/>
+
+You can generate these URLs via Web Designer's App Launch Link Dialog. (Deploy > Development > 3 Dot Menu > Get Links)
+
+<img
+    src={useBaseUrl("img/web-workflow-lifecycle-launch-link-1.png")}
+/>
+
+Use the generated link to run the specified workflow at startup.
+
+<img
+    src={useBaseUrl("img/web-workflow-lifecycle-launch-link-2.png")}
+/>
+
+# Event Service
+
+You can configure a Workflow to run on any VertiGIS Studio Web [Event](https://developers.vertigisstudio.com/docs/Web/api-commands-operations-events/#events) by registering an event listener in the [Event service](https://developers.vertigisstudio.com/blog/2023/03/01/mobile-generic-event-listener/).
+
+:::danger Warning: In general configuring an event listener for a Workflow that contains alerts or forms can be undesirable, special consideration should also be given to the performance of a Workflow that is triggered by an Event that is likely to run often as this could degrade the user experience.
+:::
+
+# Different ways to launch a new Workflow. When to use which and why?
+
+## Run Workflow Command (workflow.run)
+
+Used to start a Workflow on user activity, often executed from IWTM, menu, or button component. This is the most common way to launch a Workflow; you will generally use a variant of this command unless you need to utilize a result from the Workflow.
+
+Variants: Run Asynchronously, use this flag if want the command to return immediately and not wait on the Workflow to finish. This may be useful if your workflow.run is in a command chain that you want to continue to completion without waiting for the Workflow to finish.
```

Comment:
```suggestion
Variants: Run Asynchronously, use this flag if you want the command to return immediately and not wait on the Workflow to finish. This may be useful if your workflow.run is in a command chain that you want to continue to completion without waiting for the Workflow to finish.
```